### DISCUSSION TITLE
What is the difference between the 'edit db' and the 'installed' database.?

### DISCUSSION THREAD

**Discussion by ktarbet** [opendcs/opendcs#517] [Q&A] [answered]
I see these terms and I just want to work with 'the' database, so I get confused.

For example I'm looking at the code for DbExport.java.    The comments for main(String args[]) mention using -i to export 'installed'  database.   However, the code doesn't seem to look at the args, and uses settings.editDatabaseLocation.

```java
		dbio = DatabaseIO.makeDatabaseIO(
			settings.editDatabaseTypeCode, settings.editDatabaseLocation);
```

**Comment by MikeNeilson** [comment, marked as answer]
There used to be a concept, or perhaps there was an attempted at a concept, of having a "editing database" and a "production database".
I think that did make some sense back in the day given the speed of different operations and things but I've never really seen it used.

I think we should slowly remove references to edit vs production vs install toward a more simple "the database".
